open_bounties/0079 Python Code Samples 2.md

```diff
@@ -0,0 +1,39 @@
+---
+ID: 0079
+Discussion: https://github.com/XRPLBounties/Proposals/discussions/79
+Category: Technical Documentation
+Status: Open
+Amount: $5500
+---
+
+# Python Code Samples 2
+
+## Overview
+
+Currently, our Python documentations have [code snippets](https://github.com/XRPLF/xrpl-py/tree/master/snippets) and [code samples](https://github.com/XRPLF/xrpl-dev-portal/tree/master/content/_code-samples) for some common usecases. However, there needs to be more examples to capture common tasks that users would want the library to do. Developers would benefit from additional relevant, high quality, well-documented examples in Python.
+
+## Details
+
+To add missing code samples, follow the instructions at the bottom of the page, under **Contribute Code Samples**.
+
+https://xrpl.org/code-samples.html
+
+## Milestones
+
+| #   | Description              | Details                                                                                                                                                                                                    | Proposed Potential Award | Open? |
+| --- | ------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------ | ----- |
+| 1   | NFT                      | Mint (or batch mint) and burn an [NFT Token](https://xrpl.org/non-fungible-tokens.html#non-fungible-tokens-overview), assign an authorized minter, transfer NFT tokens (create, cancel, and accept offer). | $1000                    | Open  |
```

Comment:
I think this should be a higher amount because this is essentially the same amount of code as our javascript quickstart guide.

Reply:
1500 seems reasonable?

Reply:
Ye, 1500-2000 is reasonable.